src/LV2_Plugin/YoshimiLV2Plugin.cpp

```diff
@@ -379,7 +379,7 @@ bool YoshimiLV2Plugin::init()
 }
 
 
-LV2_Handle	YoshimiLV2Plugin::instantiate (const struct _LV2_Descriptor *desc, double sample_rate, const char *bundle_path, const LV2_Feature *const *features)
+LV2_Handle	YoshimiLV2Plugin::instantiate (const struct LV2_Descriptor *desc, double sample_rate, const char *bundle_path, const LV2_Feature *const *features)
```

Comment:
Nice cleanup, but you have to lose the `struct` as well, here and the other places where it was changed. On my computer this doesn't compile because `LV2_Descriptor` is a typedef.

Reply:
Indeed, thanks! Also removed those `struct`s.